definitions/ext-rf_scanner/summary_metrics.yml

```diff
@@ -44,10 +9,25 @@ totalTime:
     eventId: entity.guid
     
 actions:
-  title: Actions 
+  title: Volume 
   unit: COUNT
   query:
-    select: count(action)
-    from: THD_RF_CONNECTION_EVENT
+    select: count(*)
+    from: THD_RF_TRANSACTION
     eventId: entity.guid
-    
+
+errors:
+  title: Errors
+
+  query:
+    select: count(message)
+    from: Log
+    where: message like '%error%' or message like '%Error%' and message NOT LIKE '%lqqqq%' or message NOT LIKE '%Winsock error 10042%'
+
+lastactive:
+  title: Last Active Time
+  unit: STRING
+  tag:
+    key: lastActiveTimestamp
```

Comment:
This tag doesn't exist in the entity definition

Reply:
It will be present if in the telemetry exists as `tag.lastActiveTimestamp` which I don't know if it's the case

Reply:
Could you confirm if this is the case @65pony? otherwise this column will always be empty